.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +13,9 @@ Please select what type of pull request this is:
 **Related issue**
 Please link the issue your pull request is referring to. If this pull request fully resolves the relevant issue, put "closes" before the issue number. Example: "closes #123456".
```

Comment:
we probably need to change the example here also but im not very fond of the # being the only thing there